frontends/api/src/test-utils/factories/learningResources.ts

```diff
@@ -180,7 +181,7 @@ const _learningResourceShared = (): Partial<
     image: learningResourceImage(),
     offered_by: maybe(learningResourceOfferor) ?? null,
     platform: maybe(learningResourcePlatform) ?? null,
-    prices: [0.0],
+    prices: "",
```

Comment:
Is this change intentional?

Reply:
Reverted to the `[0.0]` - I changed this because the typecheck was failing for some reason (not realizing the model change that had happened inadvertently).